momento-sdk/src/test/java/momento/sdk/responses/storage/GetResponseTest.java

```diff
@@ -55,7 +55,7 @@ public void testConvenienceMethodsOnGetResponse() {
     // TODO distinguish store not found from key not found
     GetResponse.Error error =
         new GetResponse.Error(
-            new NotFoundException(
+            new CacheNotFoundException(
```

Comment:
Shouldn't this be `StoreNotFoundException` now?

Reply:
Thank you. Updated in [a85b645](https://github.com/momentohq/client-sdk-java/pull/362/commits/a85b6455e36023ef952326b93ca017eab67ed6cd)